lib/api/apiUtils/object/objectLockHelpers.js

```diff
@@ -0,0 +1,38 @@
+const moment = require('moment');
+
+/**
+ * isObjectLocked - checks whether object is locked or not
+ * @param {obect} bucket - bucket metadata
+ * @param {object} objectMD - object metadata
+ * @param {array} headers - request headers
+ * @return {boolean} - indicates whether object is locked or not
+ */
+function isObjectLocked(bucket, objectMD, headers) {
+    if (bucket.isObjectLockEnabled()) {
```

Comment:
If you swap the condition it can be a good guard clause and not require indenting